src/vstest.console/Processors/ListTestsTargetPathArgumentProcessor.cs

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.CommandLine.Processors
+{
+    using System;
+    using System.Diagnostics.Contracts;
+    using System.Globalization;
+    using Microsoft.VisualStudio.TestPlatform.CommandLine;
+    using CommandLineResources = Microsoft.VisualStudio.TestPlatform.CommandLine.Resources.Resources;
+
+    /// <summary>
+    //  An argument processor that allows the user to specify filter to run sub set of tests.
```

Comment:
Please correct the summary string. It is not conveying the actual use of this argument processor. Add a detailed comment about how this ListTestsTargetPath is going to use.

Reply:
Done